storage_service/locations/tests/test_package.py

```diff
@@ -370,13 +370,36 @@ def test_replicate_aip(self):
         )
 
         aip.create_replicas()
-        replica = aip.replicated_package
 
+        assert aip.replicas.count() == 1
+
+        replica = aip.replicas.first()
         assert replica is not None
-        # The relationship is a little bit broken here; one would think that the
-        # _original_ AIP should have replicas, and the replicated AIP should not.
```

Comment:
Turns out, the relationship was on the right side, the `create_replicas` method was just modifying `aip` in place.